### ISSUE TITLE
Has anyone experienced download issues using git lfs?

### ISSUE THREAD

**Issue by pszelew** [NVIDIA/Dataset_Synthesizer#56]
Hi,
Today I wanted do download Synthetizer and I experienced some issues. Almost all downloaded  files have size of 1kb and I'm unable to read them. Are my issues related to this link? https://help.github.com/en/github/managing-large-files/about-storage-and-bandwidth-usage
Should I wait whole month to download it again?

Data from Git Bash:

$ git lfs clone https://github.com/NVIDIA/Dataset_Synthesizer
WARNING: 'git lfs clone' is deprecated and will not be updated
          with new flags from 'git clone'
'git clone' has been updated in upstream Git to have comparable
speeds to 'git lfs clone'.
Cloning into 'Dataset_Synthesizer'...
remote: Enumerating objects: 4128, done.
remote: Total 4128 (delta 0), reused 0 (delta 0), pack-reused 4128
Receiving objects: 100% (4128/4128), 6.32 MiB | 7.99 MiB/s, done.
Resolving deltas: 100% (1954/1954), done.
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
error: failed to fetch some objects from 'https://github.com/NVIDIA/Dataset_Synthesizer.git/info/lfs'

**Comment by find1dream** [comment]
Me too

**Comment by ryoppippi** [comment]
Me toooooooooo

**Comment by TThreadgold** [comment]
Also unable to clone, is anyone able to provide a mirror of the current  (or even an outdated)  version of the repository?

**Comment by IbrahimMCode** [comment]
Was someone able to find a solution or something else?

**Comment by cksdn4872** [comment]
same problem... please fix

**Comment by avinashsen707** [comment]
Ensure LFS is installed: [(https://help.github.com/articles/installing-git-large-file-storage/.)](url)

then initialize the git LFS

then follow :
Install the NDDS files for the dataset synthesizer.

`git clone https://github.com/NVIDIA/Dataset_Synthesizer.git`

**Comment by pszelew** [comment]
> Ensure LFS is installed: [(https://help.github.com/articles/installing-git-large-file-storage/.)](url)
> 
> then initialize the git LFS
> 
> then follow :
> Install the NDDS files for the dataset synthesizer.
> 
> `git clone https://github.com/NVIDIA/Dataset_Synthesizer.git

It doesn't work as we want to. "git clone https://github.com/NVIDIA/Dataset_Synthesizer.git" doesn't download LFS files at all. Using "git lfs clone https://github.com/NVIDIA/Dataset_Synthesizer.git" unfortunately doesn't download them either. But it should :(

**Comment by ryoppippi** [comment]
Probably reached to the downloading limit?

**Comment by patrickESM** [comment]
Same problem here. tried git clone with flags and git lfs clone.

are the files missing essential for running NDDS?

**Comment by KatharinaSchmidt** [comment]
I have the same problem. 
Did someone solve it yet?

**Comment by 50618861** [comment]
Same issue...

**Comment by hanlinniu** [comment]
same problem here

**Comment by yyc9268** [comment]
I got a reply that they are planning to move this repo to gitlab.
However, it seems not decided yet.
[Check this link](https://github.com/NVlabs/Deep_Object_Pose/issues/102#issuecomment-607981066)

**Comment by mintar** [comment]
@jtremblay : Can anything be done about this?

**Comment by hanlinniu** [comment]
Does anyone have already downloaded? Would you please send a dropbox link to us?

**Comment by t-kanne** [comment]
dropbox link would be awesome...

**Comment by thangt** [comment]
Sorry for the trouble. I just posted an update and attach the compressed code there: https://github.com/NVIDIA/Dataset_Synthesizer/releases/download/1.2.2/ndds_1.2.2.zip.
Please try it out and let me know if you have any other troubles with it!

**Comment by belorenz** [comment]
> Sorry for the trouble. I just posted an update and attach the compressed code there: https://github.com/NVIDIA/Dataset_Synthesizer/releases/download/1.2.2/ndds_1.2.2.zip.
> Please try it out and let me know if you have any other troubles with it!

Thanks I could download the file! 

This repo here can't still be cloned though.

**Comment by csingh27** [comment]
Download it directly man ! @belorenz

**Comment by patrickESM** [comment]
> Download it directly man ! @belorenz

No need to be so harsh @csingh27 He just stated what's true. Would still be nice to get the code without searching the issues for a solution.

**Comment by csingh27** [comment]
@patrickESM I totally agree that the owner needs to fix this thing on the latest commit. 
Let's see when this issue gets fixed. Meanwhile I am just downloading and trying to understand it.
Just wondering though how you thought I was harsh because I just stated something normally.

**Comment by belorenz** [comment]
> Download it directly man ! @belorenz

Please read my comment one more time please.

**Comment by csingh27** [comment]
Fine dude fine @belorenz !
